mender-deb-package

```diff
@@ -149,12 +156,18 @@ build_packages() {
     sign_flags="--sign-key=$key_id"
   fi
 
+  # dh_make needs a name of the user in the env
+  export USER=root
+  # needed to get the orig file, returns 1 because debian directory exists,
+  # hence the || true, without orig file buildpackage command would fail
+  dh_make --createorig --packagename "${DEB_PACKAGE}_${DEB_VERSION_NO_REV}" -s -y || true
+
   case "$ARCH" in
     amd64)
       # Native build (amd64)
       dpkg-buildpackage \
         ${sign_flags} \
-        --build=$DEB_BUILD_TYPE
+        --build=source,$DEB_BUILD_TYPE
```

Comment:
This will create source tarballs also for commercial packages.

Although we can then handle it in the publish stem to make sure we don't publish them anywhere... I think is best to just not build them and make sure this way that we don't publish them by mistake.

With that in mind, here we should revert to use `$DEB_BUILD_TYPE`, and in [here](https://github.com/mendersoftware/mender-dist-packages/blob/master/docker-build-package#L120) prepend `source,` to the variable for non commercial packages.

What do you think?

Reply:
makes sense. thank you. fixed.